test/test_torch.py

```diff
@@ -17747,6 +17765,9 @@ def inner(self, device, dtype):
     ('acos', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types, [torch.bfloat16]),
     ('asin', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types, [torch.bfloat16]),
     ('atan', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types, [torch.bfloat16]),
+    ('acosh', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types2, [torch.bfloat16]),
+    ('asinh', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types2, [torch.bfloat16]),
+    ('atanh', '', _small_3d, lambda t, d: [], 1e-3, 1e-2, 1e-5, _float_types2, [torch.bfloat16]),
```

Comment:
Separate complex tests in `test/test_autograd.py` are added here: https://github.com/pytorch/pytorch/pull/38399. Should I wait for that to be merged?

Reply:
cc @anjali411

Reply:
I think it's worth adding `_complex_and_float_types2` since there are a bunch of tests in that lest we would eventually wanna test for both `[torch.float, torch.double]` as well as `[torch.cfloat, torch.cdouble]`

Reply:
Makes sense. Thanks @anjali411 - I've added the tests here (https://github.com/pytorch/pytorch/pull/38388/commits/76af72d98bcf56bbc5399304232baeea6b546382). When #38400 is merged, I'll replace `torch.cfloat, torch.cdouble` with `_complex_types`.